docs/source/pillars/data_management.md

```diff
@@ -149,6 +149,17 @@ The ability of the TRE operator to ensure outputs are safely published and share
     Results will be produced which address the project's purpose, some of which may be outputs that are removed from the TRE.
     Understanding what these outputs are likely to be and their sensitivity as early as possible will help prepare for their processing and publication.
   - Recommended
+* - 3.3.3.
+  - You must have a process for disclosure control that ensures outputs are safe.
+  - This process should define expected risks and how to mitigate them.
+    All TRE outputs must be subject to this process.
+    You might choose to follow existing guidelines, for example around statistical disclosure.
```

Comment:
I would expect "for example" to be followed by a specific example.
Is there one we want to draw attention to?

Otherwise, maybe we can just say "You might choose to follow existing guidelines"?

Reply:
With ML models you cannot guarantee outputs are safe so need to change the wording.

Reply:
How about? "You must have a documented process for disclosure control of output files from the TRE."

Reply:
> With ML models you cannot guarantee outputs are safe

@drchriscole do you think we also need a statement regarding outputs that cannot be easily determined as safe, such as ML models

Something like:

- Statement: You must have a documented policy for handling disclosure risks associated with outputs that cannot be manually checked
- Guidance: It is generally not recommended to egress TRE outputs that cannot be manually checked, but in cases such as when you want to egress a machine learning model, there should be an institutional policy about what to do if such a model leaks data that was kept secure whilst it was inside the TRE
- Mandatory

Reply:
How about

Guidance: It is generally recommended that only files that can be checked are egressed. For example, it is difficult to check for the presence of sensitive data in binary files or very large numeric files. If egress is allowed the risks of inadvertent disclosure must be mitigated and documented.

Reply:
See 3.3.5 just added